es_ES/faq.md

```diff
@@ -0,0 +1,625 @@
+un [Padding](https://docs.flutter.io/flutter/widgets/Padding-class.html),
+un [Center](https://docs.flutter.io/flutter/widgets/Center-class.html),
+un [Material](https://docs.flutter.io/flutter/material/Material-class.html),
+un [AnimatedDefaultTextStyle](https://docs.flutter.io/flutter/widgets/AnimatedDefaultTextStyle-class.html), y un [ConstrainedBox](https://docs.flutter.io/flutter/widgets/ConstrainedBox-class.html).
+El [InkWell](https://docs.flutter.io/flutter/material/InkWell-class.html)
+esta construido usando un [GestureDetector](https://docs.flutter.io/flutter/widgets/GestureDetector-class.html).
+El [Material](https://docs.flutter.io/flutter/material/Material-class.html)
+esta construido usando un [AnimatedDefaultTextStyle](https://docs.flutter.io/flutter/widgets/AnimatedDefaultTextStyle-class.html),
+un [NotificationListener](https://docs.flutter.io/flutter/widgets/NotificationListener-class.html),
+y un [AnimatedPhysicalModel](https://docs.flutter.io/flutter/widgets/AnimatedPhysicalModel-class.html).
+Y así. Son widgets todo el camino.
+
+* Programación funcional: las aplicaciones completas pueden construirse solo con
+[StatelessWidget](https://docs.flutter.io/flutter/widgets/StatelessWidget-class.html), que son esencialmente funciones que describen cómo los argumentos se asignan a otras funciones, cayendo en primitivas que computan layouts o gráficos de pintura. (Tales aplicaciones no pueden tener estado fácilmente, por lo que normalmente no son interactivas.)
+Por ejemplo, el [Icon](https://docs.flutter.io/flutter/widgets/Icon-class.html)
+widget es esencialmente una función que mapea sus argumentos
+([color](https://docs.flutter.io/flutter/widgets/Icon/color.html),
+[icon](https://docs.flutter.io/flutter/widgets/Icon/icon.html),
+[size](https://docs.flutter.io/flutter/widgets/Icon/size.html)) dentro de layout
+primitivos. Además, se hace un uso intensivo de estructuras de datos inmutables
+incluyendo toda la jerarquía de clase de
+[Widget](https://docs.flutter.io/flutter/widgets/Widget-class.html), así como numerosas clases de apoyo tales como
+[Rect](https://docs.flutter.io/flutter/dart-ui/Rect-class.html) y
+[TextStyle](https://docs.flutter.io/flutter/painting/TextStyle-class.html). En una menor escala, La API
+[Iterable](https://docs.flutter.io/flutter/dart-core/Iterable-class.html) de Dart,
+que hace un uso intensivo del estilo funcional (map, reduce, where, etc.), es
+frecuentemente utilizado para procesar listas de valores en el framework.
+
+* Programación event-driven: las interacciones del usuario están representadas por objetos de eventos que se envían a callbacks registradas con controladores de eventos. Actualizaciones de pantalla son activados por un mecanismo de callback similar. La clase [Listenable](https://docs.flutter.io/flutter/foundation/Listenable-class.html), que se utiliza como base del sistema de animación, formatea un modelo de suscripción para eventos con múltiples oyentes.
```

Comment:
formalises a subscription model for events with multiple listeners.
formaliza un modelo de suscripción para eventos con múltiples oyentes